source/reference/auth-plugin-jdbc.txt

```diff
@@ -14,18 +14,20 @@ JDBC Authentication Plugin
 
 .. _install-auth-jdbc-plugin:
 
-.. important::
+Prerequisites
+-------------
 
-   The plugin is not compatible with MySQL Server or Connector/J
-   driver version 8 and later.
+This plugin:
+
+- Requires :mvn:`MySQL Connector/J 5.1.39 </mysql/mysql-connector-java/5.1.39>` or later.
```

Comment:
`or later` contradicts the statement in the next bullet. Consider describing the version supported in the first bullet (greater than 5.1.39 and less than 8) and describe the MySQL server requirements in the second bullet.